src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -206,8 +227,16 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
         subset_dir (str):
             The name of the directory containing the subsetted pixel data.
             Should be placed in `pixel_output_dir`.
-        norm_vals_name (str):
-            The name of the file to store the 99.9% normalization values
+        norm_vals_name_pre_rownorm (str):
+            The name of the file to store the pre pixel-thresholded norm values
```

Comment:
For clarity, can we change this to "The name of the file to store the pre pixel-normalization channel norm values"